ntbs-service/Services/UserService.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+using ntbs_service.Models;
+
+namespace ntbs_service.Services
+{
+  public interface IUserService
+  {
+    Task<List<TBService>> TbServices(ClaimsPrincipal user);
+    UserType GetUserType(ClaimsPrincipal user);
+  }
+
+  public class UserService : IUserService
+  {
+    private readonly AdfsOptions config;
+    private readonly NtbsContext context;
+    
+    public UserService(NtbsContext context, IOptionsMonitor<AdfsOptions> options) {
+      this.context = context;
+      config = options.CurrentValue;
+    }
+    async public Task<List<TBService>> TbServices(ClaimsPrincipal user)
```

Comment:
By convention think this would rather be called Task< `GetTbServicesAsync` and should probably have the method declaration be for `Task<IEnumerable...>` rather than the concrete `List`